views/machines/index.html

```diff
@@ -70,20 +70,29 @@
 <div class="row">
     <div class="col-md-12">
         <h1 class="page-header">Machines
-            <button type="button" class="btn btn-green pull-right" onclick="createVMdialog()"><span class="glyphicon glyphicon-plus" style="font-size:15px"></span>&nbsp;&nbsp;New Machine</button>
+            <button id="newMachine" type="button" class="btn btn-green pull-right" onclick="createVMdialog()" disabled=""><span class="glyphicon glyphicon-plus" style="font-size:15px"></span>&nbsp;&nbsp;New Machine</button>
         </h1>
         <div id="error" class="alert alert-danger text-center" role="alert" style="display:none">
             <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span> <span id="errormessage"></span>
             <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
         </div>
        
-        <div>
+        <div style="height: 34px">
+        <span id="projectChoiceBlock">
             Current Project:
-            <select id="projectChoice" class="btn btn-default btn-xs" onchange="makeAjaxCalls()"></select>
+            <select style="height:34px" id="projectChoice" class="btn btn-default btn-xs" onchange="makeAjaxCalls()"></select>
             <span id="loadingStatus"></span>
+        </span>
+
+        <span id="SearchBlock" style="float:right; height:auto;">
+<!--
+            Search:
+            Ideally the search box should go here but i can't seem find it anywhere in this repo
```

Comment:
Was there a search box before?

Reply:
So, what's happening here?

Reply:
I believe the search bar element is created by jquery in one of the min.js files ([this one i think](https://github.com/stfc/cloud/blob/master/assets/components/jquery/jquery.dataTables-1.10.10.min.js)). This is near impossible to read, even after de-minifying it.

The element also does not have any IDs or unique classes which makes it very difficult to modify with JQuery and CSS.

The search bar is functional but is placed in a different location and under different parents to the other elements next to it (ie.the filter button and projects drop down). I'm not sure why this is and it is awkward for CSS.

Reply:
Ok, sounds like you should record that as an issue.